src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/Semantic/DefaultRazorSemanticTokenInfoServiceTest.cs

```diff
@@ -278,68 +263,6 @@ public async Task GetSemanticTokens_CSharp_StaticModifierAsync()
 
             await AssertSemanticTokensAsync(txt, isRazor: false, csharpTokens: cSharpResponse, documentMappings: mappings);
         }
-
-        [Fact]
-        public async Task GetSemanticTokens_CSharp_UsesCache()
```

Comment:
We definitely still want some tests of the Cache behavior.

Reply:
I think we do too, but I believe we previously tested by seeing how many times a handler was called in relation to one another. imo this becomes much harder with only a single handler. I think writing proper tests for this may be easier once we move away from Moq, I can include it as part of https://github.com/dotnet/razor-tooling/issues/5948. Or we might eventually just need to include a "test-only" return param to indicate whether the cache was used, but since QB mode ends soon I will do it in a follow-up PR.